### PR TITLE
Add obstacles to 2-DOF manipulator model

### PR DESCRIPTION
**Pull request by sea-bass**
This.

image

### CODE REVIEW

File: src/pyroboplan/models/two_dof.py [hunk 1]
```diff
@@ -20,3 +23,46 @@ def load_models():
...
+
+
+def add_object_collisions(model, collision_model, visual_model):
+    """
+    Adds obstacles and collisions to the 2-DOF manipulator collision model.
+
+    Parameters
+    ----------
+        model : `pinocchio.Model`
+            The Panda model.
```
Comment (eholum) [inline src/pyroboplan/models/two_dof.py:35]: Lol this doesn't really matter but s/Panda/2-DOF/

**Review by eholum — APPROVED** [review]
nit? kind of nit.

File: src/pyroboplan/models/two_dof.py [hunk 2]
```diff
@@ -20,3 +23,46 @@ def load_models():
...
+        collision_model : `pinocchio.Model`
+            The Panda collision geometry model.
+        visual_model : `pinocchio.Model`
+            The Panda visual geometry model.
```
Comment (sea-bass) [inline src/pyroboplan/models/two_dof.py:39]:
```suggestion
        model : `pinocchio.Model`
            The robot model.
        collision_model : `pinocchio.Model`
            The collision geometry model.
        visual_model : `pinocchio.Model`
            The visual geometry model.
```

File: src/pyroboplan/models/two_dof.py (hunk 1 above)
Comment (sea-bass) [reply src/pyroboplan/models/two_dof.py]: eagle eye!

**Pull request merged**